### ISSUE TITLE
Weird hash

### ISSUE THREAD

**Issue by minderfolden** [urbit/urbit#4440]
**Describe the bug**
My star has a weird hash, and my planet got it too. I searched in here and in the dev email list but I couldn't find anything. Tried merging with the star's sponsor `~bus`  and the same hash remained. 

**To Reproduce**
Steps to reproduce the behaviour:
1. Woke up to this, so I don't know.

**Expected behaviour**

**Screenshots**
Red is the star
![image](https://user-images.githubusercontent.com/51895363/107854445-713fae00-6de1-11eb-8eee-92c607f788ec.png)

Green is my planet
![image](https://user-images.githubusercontent.com/51895363/107854455-93d1c700-6de1-11eb-9a8c-6d538cb77f47.png)

**System (please supply the following information, if relevant):**
 - OS: raspbian for both
 - Vere and Urbit OS versions Running the Pi binaries built by ~botter
 - Your ship's `%base` hash (use `+trouble` to check) Star: lv7cq Planet: 69113

**Additional context**
I have a group with 100+ people in it.

**Notify maintainers**
If you happen to know who the appropriate maintainers are, consider mentioning them with an @ here.  You may want to use `git blame` to see who has last touched any relevant code.

**Comment by botter-nidnul** [comment]
Sounds like this https://github.com/urbit/urbit/issues/3932#issuecomment-742877047

**Comment by philipcmonk** [comment]
I agree with @botter-nidnul, those commands should fix it.  Does that solve it for you?
